notify-send.sh

```diff
@@ -78,9 +78,9 @@ make_action() {
 
 make_hint() {
     type=$(convert_type "$1")
-    [[ ! $? = 0 ]] && return 1
+    [ $? -ne 0 ] && return 1
```

Comment:
Good place to start:  conditionals in POSIX shell doesn't use double brackets, and the subset of things one is allowed to do is more limited as well.  Instead, single brackets are used, which are equivalent to `test(1p)` command.

Here, specifically, we replace `[[ ! ... = ... ]]` with a numeric comparison `[ ... -ne ... ]` since we're testing the value of `$?`.